changelog/logger_sharedLog_returning_shared_logger.dd

```diff
@@ -0,0 +1,4 @@
+Change sharedLog to return shared(Logger)
```

Comment:
Change `module.path.of.sharedLog` to return shared(Logger)

We shouldn't assume the reared of the changelog is intimately familiar with the layout of the standard library.